examples/app-pytorch/client_low_level.py

```diff
@@ -20,16 +20,16 @@ def hello_world_mod(msg, ctx, call_next) -> Message:
 @app.train()
 def train(msg: Message, ctx: Context):
     print("`train` is not implemented, echoing original message")
-    return msg.create_reply(msg.content, ttl="")
+    return msg.create_reply(msg.content, ttl=DEFAULT_TTL)
```

Comment:
Will this be the future use case? I mean, import `DEFAULT_TTL` and use it? Should we set the default value for `ttl` in `Message.create_reply`?

Reply:
I added `DEFAULT_TTL` as the default value, set to 1h should be fine for most cases.